vertical-pod-autoscaler/pkg/utils/status/status_object.go

```diff
@@ -57,6 +57,11 @@ type Client struct {
 	holderIdentity       string
 }
 
+// Validator for the status object.
+type Validator interface {
+	IsStatusValid(leaseTimeout time.Duration) (bool, error)
```

Comment:
can you change leaseTimeout to something more implementation neutral? 
It doesn't have to be a lease backing the status.

Reply:
Done. `statusTimeout` it is.